src/sparseml/exporters/transforms/gemm_to_matmulinteger_add_cast_mul.py

```diff
@@ -44,36 +39,35 @@ class GemmToMatMulIntegerAddCastMul(OnnxTransform):
     are not necessarily quantized into a MatMulInteger followed by
     a bias add and cast to FP32
 
+    Transforms
+    ```
+    |     weight (intializer)
+    |        |
+    | input  Q
+    |   |    |
+    |   Dq   Dq  bias (initializer)
+    |     |  |  |
+    |       Gemm
     ```
-    | Starting with:
-    |
-    |                       weight (initializer)
-    |                         |
-    |          INPUT        QuantizeLinear
-    |            |            |
-    |     DequantizeLinear  DequantizeLinear   bias (initializer)
-    |                  |      |              /
-    |                       Gemm
-    |                         |
-    |                       OUTPUT
-    | We end up converting to:
-    |       INPUT
-    |         |
-    |     MatMulInteger (with constant uint8 kernel)
-    |         |
-    |     Add (constant bias + zero point correction)
-    |         |
-    |     Cast (INT32 -> FP32)
-    |         |
-    |     Mul (Rescale from bias scale)
-    |         |
-    |       OUTPUT
+    (where `Q` is QuantizeLinear, and `Dq` is DequantizeLinear)
+
+    ```
+    |   input
+    |     |
+    | MatMulInteger (with constant uint8 kernel)
+    |     |
+    | Add (constant bias + zero point correction)
+    |     |
+    | Cast (INT32 -> FP32)
+    |     |
+    | Mul (Rescale from bias scale)
     ```
     """
 
     def transform(self, model: ModelProto) -> ModelProto:
+        graph = ONNXGraph(model)
         matches = get_structural_matches(
-            ONNXGraph(model),
+            graph,
             op_type="Gemm",
             parent_ops=[
                 ["DequantizeLinear"],
```

Comment:
This should be `any_of("QuantizeLinear", "DequantizeLinear")` and update docstrings